docs/source/release_notes.rst

```diff
@@ -2,8 +2,18 @@
 
 Release Notes
 -------------
-Future Release
-==============
+.. Future Release
+  ==============
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. Thanks to the following people for contributing to this release:
+
+v1.11.0 June 30, 2022
+====================
```

Comment:
Underline needs to cover the entire text for docs to build

Reply:
changed